src/erp/payment/PaymentProcessor.java

```diff
@@ -0,0 +1,21 @@
+package erp.payment;
+
+import java.util.Scanner;
+
+public class PaymentProcessor {
+
+    public void processPayment() {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Creating a scanner object each time the function is invoked may not be efficient. Consider using dependency injection to pass the input source as a dependency.